packages/manager/cypress/support/plugins/split-run.ts

```diff
@@ -1,15 +1,18 @@
 /**
- * @file Implements naive parallelization without Cypress Cloud.
+ * @file Allows parallelization without Cypress Cloud.
  */
 
 import { globSync } from 'glob';
-
+import { resolve } from 'path';
+import { readFileSync } from 'fs';
+import { SpecWeight, SpecWeights, specWeightsSchema } from './generate-weights';
 import type { CypressPlugin } from './plugin';
 
 export const splitCypressRun: CypressPlugin = (_on, config) => {
   const splitRunEnabled = config?.env?.['CY_TEST_SPLIT_RUN'];
   const splitRunTotalRunners = config?.env?.['CY_TEST_SPLIT_RUN_TOTAL'];
   const splitRunRunnerIndex = config?.env?.['CY_TEST_SPLIT_RUN_INDEX'];
+  const splitRunWeightsPath = config?.env?.['CY_TEST_SPLIT_RUN_WEIGHTS'];
```

Comment:
Could we do something like this?
```suggestion
  const {
    CY_TEST_SPLIT_RUN: splitRunEnabled,
    CY_TEST_SPLIT_RUN_TOTAL: splitRunTotalRunners,
    CY_TEST_SPLIT_RUN_INDEX: splitRunRunnerIndex,
    CY_TEST_SPLIT_RUN_WEIGHTS: splitRunWeightsPath,
  } = config.env;
```